pkg/netutil/netutil.go

```diff
@@ -49,37 +49,7 @@ type CNIEnv struct {
 
 type CNIEnvOpt func(e *CNIEnv) error
 
-func UsedNetworks(ctx context.Context, client *containerd.Client) (map[string][]string, error) {
-	nsService := client.NamespaceService()
-	nsList, err := nsService.List(ctx)
-	if err != nil {
-		return nil, err
-	}
-	used := make(map[string][]string)
-	for _, ns := range nsList {
-		nsCtx := namespaces.WithNamespace(ctx, ns)
-		containers, err := client.Containers(nsCtx)
-		if err != nil {
-			return nil, err
-		}
-		nsUsedN, err := namespaceUsedNetworks(nsCtx, containers)
-		if err != nil {
-			return nil, err
-		}
-
-		// merge
-		for k, v := range nsUsedN {
-			if value, ok := used[k]; ok {
-				used[k] = append(value, v...)
-			} else {
-				used[k] = v
-			}
-		}
-	}
-	return used, nil
-}
-
-func namespaceUsedNetworks(ctx context.Context, containers []containerd.Container) (map[string][]string, error) {
+func NamespaceUsedNetworks(ctx context.Context, containers []containerd.Container) (map[string][]string, error) {
```

Comment:
I think you just make `NamespaceUsedNetworks` public here, I think it's necessary to reorder the function